ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -1204,6 +1203,18 @@ private static void moveSpecifiedFiles(FileSystem fs, Path src, Path dst, Set<Pa
     }
   }
 
+  public static void moveSpecifiedFiles(FileSystem fs, Path dst, Set<Path> filesToMove)
+      throws IOException, HiveException {
+    if (!fs.exists(dst)) {
+      fs.mkdirs(dst);
```

Comment:
Can you check mkdirs() contract. I think if dst exists, it will return silently. i.e., no need to check for exists() first.

Reply:
This was a copy paste from existing code. I'll check if exists is required or not.